change/@fluentui-react-charting-4a2b4230-b5e3-4cea-9777-4e1670b33156.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33509&commitId=b255427193e98bd1b5fd266257c3f602aa86bdc7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic.chromium.png|307| Changed|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|320| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Change in VR screenshot as per changes in the PR.